frontend/src/components/feature/chat/ChatMessage/Renderers/FileMessage.tsx

```diff
@@ -54,40 +59,46 @@ export const FileMessageBlock = memo(({ message, user, ...props }: FileMessageBl
 
             </video>
         </Flex> :
-
-
+            
             <Flex
-                align='center'
+                direction="column"
                 gap='4'
                 p='4'
+                height="220px"
+                maxWidth={isDesktop ? '320px' : '100%'}
                 className="border-2 bg-gray-2 dark:bg-gray-4 rounded-md border-gray-4  dark:border-gray-6 shadow-sm">
-                <Flex align='center' gap='2'>
-                    <FileExtensionIcon ext={fileExtension} />
-                    <Text as='span' size='2' className="text-ellipsis overflow-hidden line-clamp-1">{fileName}</Text>
-                </Flex>
+            
+                            
+                <img src={test} width="100%" height="80%"/>
```

Comment:
Remove the hardcoded test image and replace it with a more appropriate placeholder or loading state